packages/react-core/src/components/TimePicker/TimePicker.tsx

```diff
@@ -221,7 +226,27 @@ export class TimePicker extends React.Component<TimePickerProps, TimePickerState
   };
 
   scrollToIndex = (index: number) => {
-    this.getOptions()[index].offsetParent.scrollTop = this.getOptions()[index].offsetTop;
+    // fixes issue where inline value for menutAppendTo results in initial menu item that is scrolled to being out of view
+    let indexToScroll = index;
+    if (this.props.menuAppendTo === 'inline') {
+      if (indexToScroll > 0) {
+        indexToScroll = index - 1;
+      } else {
+        indexToScroll = 0;
+      }
+    }
```

Comment:
Nit: I think this would be a bit cleaner/simpler with a nonmutation implementation that puts this logic in its own function rather than having a nested if/else that mutates a variable.